packages/backend/server/src/modules/payment/stripe.ts

```diff
@@ -0,0 +1,18 @@
+import { FactoryProvider } from '@nestjs/common';
+import { omit } from 'lodash-es';
+import Stripe from 'stripe';
+
+import { Config } from '../../config';
+
+export const StripeProvider: FactoryProvider = {
+  provide: Stripe,
+  useFactory: (config: Config) => {
+    const stripeConfig = config.payment.stripe;
+
+    return new Stripe(
+      stripeConfig.keys.APIKey,
```

Comment:
does the server not consider the scene where payment is not enabled? e.g. self hosted

Reply:
merge and fix later, if have some problem